docs/Documentation/Environment/Building_Packages/modules.md

```diff
@@ -5,66 +5,84 @@ parent: Building Packages
 
-2. Now logout, log back in, and you should see your personal modules collection with a brand new module.
+In the `openmpi/4.1.0/gcc840` directory you just made, or whatever directory name you chose, copy the actual modulefile. 
 
+It is much easier to copy an example from the system collection than to write one _de novo_. To copy the system modulefile, run:
+
+???+ example "On Eagle"
 	```
-	[$USER@el1 ~]$ module avail
-	
-	---------------------------------- /scratch/$USER/modules/default -----------------------------------
-	openmpi/4.1.0/gcc-8.4.0
+	cp /nopt/nrel/apps/modules/default/modulefiles/openmpi/4.0.4/gcc-8.4.0.lua /scratch/$USER/modules/default/openmpi/4.1.0/.
 	```
-	
-	Notice that the ".lua" extension does not appear--the converse is also true, if the extension is missing it will not appear via module commands!
-	As a sanity check, it's a good idea to load the module, and check that an executable file you know exists there is in fact on your PATH:
-	
+
+???+ example "On Eagle"
 	```
-	[$USER@el1 ~]$ module load openmpi/4.1.0/gcc-8.4.0
-	[$USER@el1 ~]$ which mpirun
-	/scratch/$USER/openmpi/4.1.0-gcc-8.4.0/bin/mpirun
+	cp /nopt/nrel/apps/modules/default/compilers_mpi/openmpi/4.1.5-gcc /scratch/$USER/modules/default/openmpi/4.1.0/.
 	```
 
+???+ warning "OpenMPI modulefile on Kestrel"
+	Please note that the OpenMPI modulefile on Kestrel is of TCL type
+	It is not necessary for you to know the language to modify our examples.
+	
+	The Lmod modules system uses the Lua language natively for module code. Tcl modules will also work under Lmod, but don't offer quite as much flexibility.
+## Edit the modulefile copy as needed
+For this example, (a) the OpenMPI version we're building is 4.1.0 instead of 4.0.4 on Eagle or 4.1.5 on Kestrel, and (b) the location is in `/scratch/$USER`, rather than `/nopt/nrel/apps`. 
+
+So, edit `/scratch/$USER/modules/default/openmpi/4.1.0/gcc-8.4.0.lua` to make the required changes. 
+
+Most of these changes only need to be made at the top of the file; variable definitions take care of the rest.
+
+## Add your new modulefile location to a login script
+In order to add a location to be searched regularly for available modules, the module use command may be added to a login script (e.g., ```.bash_profile```) or issued in an interactive shell or job script. In your `$HOME/.bash_profile`, add the following line near the top:
+
+```
+module use -a /scratch/$USER/modules/default
+```
+
+The -a flag appends the path that follows to environment variable MODULEPATH; leaving it out will prepend the path. The first module found in searching $MODULEPATH is used, so the search order is important.
+
+If you've built packages before and enabled them this way, you don't have to do this again!
+
+By changing ```$HOME``` in the instructions above to a project location (e.g., ```/projects/<your project name>```), you can create module collections that all users on a project can see and use.
```

Comment:
This line needs to be adapted to fit into this page. Which "instructions above", and the instructions aren't using $HOME.